### ISSUE TITLE
Add a very simple "leave a comment" box after donation

### ISSUE THREAD

**Issue by valianttry** [mozilla/donate.mozilla.org#908]
This is the second donor@ email I've received with this request:

**I just contributed. Wish there was a place to add a comment at time of giving money.
Consider adding a comment/feedback space. Would have added to pleasure of giving $ to say a nice comment.
For me, somehow such feedback solidifies/full-circle for contribution process. Kind-of like thanking the person ringing bell at Salvation Army kettle as i toss money in.**

How difficult would it be to put a very simple, open "Tell us why you decided to give -- leave a comment"  or "Send a message to Mozilla with your gift: " + a text box somewhere in the donor funnel?

@lovegushwa @edrushka 

Related: https://github.com/mozilla/donate.mozilla.org/issues/746 (but the survey will have 3 options for research purposes + comment)

**Comment by ScottDowne** [comment]
Reminds me of the feedback section from call congress. We can even use google forms for the data storage again if we want.

**Comment by edrushka** [comment]
I agree it would be awesome to have this - the written input we get is very valuable. My instinct is to put this on the the thank you confirmation/email signup page. To me, it strengthens the ask for their email address by signaling we're interested in a 2-way dialogue.

**Comment by ScottDowne** [comment]
I'm also curious if we can ask for feedback in cases where we display errors. If we just ask for it on the thank you page, we're going to get mostly positive responses.

**Comment by valianttry** [comment]
+1 to adding it to the Thank you... can the "submit" button work for either/or inputs? e.g. if people want to leave a comment but not sign up for email they can hit "submit" after typing their comment; and vice versa.

**Comment by ScottDowne** [comment]
@valianttry Yup, can add it to thank you separately from signup, but in the same page.

I don't know if we'll get to this though, feels very bluesky.

**Comment by valianttry** [comment]
@ScottDowne no problem, i'll bluesky it. We're getting a nice flow from the donate@ inbox, and we'll get more from the other survey we're working on.

**Comment by RileyCamila** [comment]
I have a heart for helping others.  Because I was involved with the school system, I saw first hand how under funded school systems were and how disadvantaged children did not have the necessary school supplies.  More focus was placed on getting school supplies then on educating the children. https://www.qikso.com

**Comment by RileyCamila** [comment]
I have a heart for helping others.  Because I was involved with the school system, I saw first hand how under funded school systems were and how disadvantaged children did not have the necessary school supplies.  More focus was placed on getting school supplies then on educating the children. https://www.qikso.com
